_posts/2015-02-09-index-solr-documents.markdown

```diff
@@ -47,24 +47,14 @@ GeoBlacklight provides a rake task to index documents as fixtures for tests. We
      $ rake geoblacklight:server
      ```
 
-### Index fixture metadata documents
-
-  1. Run the GeoBlacklight fixture indexer
-
-     ```sh
-     $ rake geoblacklight:solr:seed
-     ```
-    
-The GeoBlacklight Solr seed task, indexes metadata documents (in JSON format) stored in the `spec/fixtures/solr_documents` directory.
-
```

Comment:
It is not necessary to run the seed task as documents are indexed by default when starting up the server. If, however, we want people to open another terminal window to index documents with seed, we should say that explicitly.